olive/engine/packaging/packaging_config.py

```diff
@@ -22,3 +22,4 @@ class PackagingConfig(ConfigBase):
 
     type: PackagingType = PackagingType.Zipfile
     name: str = "OutputModels"
+    export_in_mlflow_format: bool = False
```

Comment:
is it possible we export other format? if yes, we might need change the name like export_format or something else?

Reply:
As for format, I don't think we have other cases for now. As for model type, even though Olive doesn't limit output model type, we only focus on onnx model now.